src/main/resources/assets/js/application/recruiter/views/tdpr-recruiter-add-interviewer.html

```diff
@@ -29,20 +29,23 @@ <h1 id="interviewer-details-h1">Add new interviewer</h1>
         <div class="form-group">
             <label class="col-md-4 control-label">E-mail:</label>
             <div class="col-md-8">
-                <input class="form-control input" type="email" ng-model="person.email" required/>
+                <input class="form-control input" name="intemail" type="email" ng-model="person.email" ng-maxlength="100" required/>
             </div>
+            <p class="int-err center" ng-show="intform.intemail.$error.maxlength">Email cannot have more than 100 characters.</p>
         </div>
         <div class="form-group">
             <label class="col-md-4 control-label">First name:</label>
             <div class="col-md-8">
-                <input class="form-control input" type="text" ng-model="person.firstName" required/>
+                <input class="form-control input" name="intfirstname" type="text" ng-model="person.firstName" ng-maxlength="35" required/>
             </div>
```

Comment:
I don't think that int prefix on input fields is needed, we have this prefix on form name already. (ex. intform.intemail )